lib/experimental/fonts/font-library/class-wp-font-family.php

```diff
@@ -482,6 +496,29 @@ private function create_font_post() {
 		return $post_id;
 	}
 
+	/**
+	 * Gets the font faces that are in both the existing and incoming font families.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $existing The existing font faces.
+	 * @param array $incoming The incoming font faces.
+	 * @return array The font faces that are in both the existing and incoming font families.
+	 */
+	private function get_intersecting_font_faces( $existing, $incoming ) {
```

Comment:
Should I add a test for this?

Reply:
I think we are not testing private class methods, and I also don't know if there is an easy way to do that and if we should do it. I would prefer to add a test case(s) for the public method (install) that covers the situations we are trying to fix with the changes in this PR.